site/docs/config-file.mdx

```diff
@@ -67,20 +67,20 @@ These plugins will run after once all presets operations are complete.
   ```js
   // cssnano.config.js
   module.exports = {
-    plugins: [require('autoprefixer')]
+    plugins: [ require('autoprefixer') ]
     
     // or
-    plugins: ['autoprefixer', 'postcss-preset-env']
+    plugins: [ 'autoprefixer', 'postcss-preset-env' ]
     
     // or
     plugins: [ 
-      ['autoprefixer', { remove: false },
+      [ 'autoprefixer', { remove: false } ],
     ]
 
     // or
     plugins: [
       [ require('autoprefixer'), {remove: false} ],
-      [ 'postcss-preset-env']
```

Comment:
These lines were the example I used for formatting the others. If the spacing for clarity seemed appropriate for the maximally complex example, it can't hurt in the simpler examples.